pkgs/games/unciv/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+, jre
+, libpulseaudio
+, libXxf86vm
+}:
+let
+  desktopItem = makeDesktopItem {
+    name = "unciv";
+    exec = "unciv";
+    comment = "Unciv, an open-source Android/Desktop remake of Civ V";
+    desktopName = "Unciv";
+    categories = "Game;";
+  };
+
+  envLibPath = stdenv.lib.makeLibraryPath [
+    libpulseaudio
+    libXxf86vm
+  ];
+
+in
+stdenv.mkDerivation rec {
+  pname = "unciv";
+
+  version = "3.9.4";
+
+  src = fetchurl {
+    url = "https://github.com/yairm210/Unciv/releases/download/${version}/Unciv.jar";
+    sha256 = "0i78mb0vby5v4hrdggfqp4d5iznixygm92y8d6hcgrf777nibb73";
+  };
+
+  phases = "installPhase";
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/opt
+    cp ${src} $out/opt/Unciv.jar
```

Comment:
Use `install`.

Reply:
Can you elaborate a bit please?

Reply:
Like this:

```nix
install -Dm644 ${src} $out/lib/Unciv.jar
```

Technically, you don't even need that, as you can use `${src}` directly in `makeWrapper`.

Reply:
You are right.